bodhi/server/models.py

```diff
@@ -818,6 +808,31 @@ def _composite_karma(self):
 
         return positive_karma, negative_karma
 
+    def get_comments_since_karma_reset(self):
```

Comment:
I recommend naming this ```comments_since_karma_reset()``` and using the [```@property```](https://docs.python.org/2/library/functions.html#property) decorator on it. Something like this:

```
@property
def comments_since_karma_reset(self):
    …
```

This will allow callers to use it like an attribute instead of having to explicitly call it, which is kind of nice. For example, callers can use ```update.comments_since_karma_reset``` instead of ```update.get_comments_since_karma_reset()```. A lot of the methods on the update class work like this, so it would fit in nicely.